app/main.py

```diff
@@ -4,4 +1,24 @@
+import os
+import requests
+from requests import Response
+
+
+def get_weather(api_key: str, city: str = "Paris") -> Response:
+    base_url = "http://api.weatherapi.com/v1/current.json"
+    params = {"key": api_key, "q": city}
+    response = requests.get(base_url, params=params)
```

Comment:
It's a good practice to handle potential exceptions that may arise from the `requests.get` call, such as network errors, by wrapping it in a try-except block.